content/docs/setup/kubernetes/install/operator/index.md

```diff
@@ -0,0 +1,118 @@
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [istioctl](/docs/ops/setup/istioctl/)
+commands and a new Operator API.
+
+{{< tip >}}
+The Istio Operator is a replacement for Helm templates and Tiller
+installation/operation.
+{{< /tip >}}
+
+## Prerequisites
+
+Before you begin, you must meet the following prerequisites.
+
+1. [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release).
+1. Perform any necessary [platform-specific setup](/docs/setup/kubernetes/platform-setup/).
+1. Check the [Requirements for Pods and
+   Services](/docs/setup/kubernetes/additional-setup/requirements/).
+
+## Install Istio in One Line
+
+The simplest option is to install Istio using a one-line command.
+
+At the command prompt, run the following command:
+
+`istioctl manifest apply`
+
+When you use this option, Istio performs the following actions:
+
+- Applies the following built-in default manifests for the following components:
+    - `Injector`
+    - `Telemetry`
+    - `Citadel`
+    - `Policy`
+    - `IngressGateway`
+    - `EgressGateway`
+    - `Galley`
+    - `CertManager`
+    - `Pilot`
+    - `NodeAgent`
+    - `crds`
+
+- Installs the default profile into the cluster.
+
+You can check if the installation succeeded by following the steps at
+[Verifying the installation](/docs/setup/kubernetes/install/kubernetes/#verifying-the-installation).
+
+## View the Contents of the Default Profile
+
+You can view the contents of the default profile by using the following command:
+
+`istioctl profile dump`
+
+## Display the List of Profiles
+
+You can install Istio using a different profile than the default by using the
+following command:
+
+`istioctl profile list`
+
+This command displays the names of Istio configuration profiles that are
+accessible by the istioctl operator commands.
+
+Optionally, you can use the `-s` flag with a install package path to see the
+list of configuration profiles available for other Istio versions.
+
+At the command prompt, run the following command:
+
+`istioctl profile list -s installPackagePath=https://github.com/istio/istio/releases/tags/1.3.3`
+
+## Select a Different Profile for Installation
+
+Any Istio configuration profile can be installed in a cluster using the
+following command:
+
+`istioctl manifest apply --set profile=<profile name>`
+
+where `<profile name>` is one of the profile names from the output of the
+`istioctl profile list` command.
+
+## Generate a Manifest to Inspect/Modify Before Installation
+
+You can inspect or modify the manifest before installing Istio using the
+following steps:
+
+1. At the command prompt, run the following command: `istioctl manifest generate > <filename>`
+1. Inspect or modify the manifest as needed.
+1. Then, run the following command: `kubectl apply -f <filename>` where `<filename>`
+   is the file name of the manifest file.
+
+{{< tip >}}
```

Comment:
callouts preceed the code that should be run.

Reply:
addressed